### ISSUE TITLE
Rails serialized attribute hash saved as Syck::PrivateType

### ISSUE THREAD

**Issue by espen** [ricardochimal/taps#90]
Field (Rails, attribute before type cast) in PostgreSQL locally: nil
Field on Heroku after exporting with taps: #<Syck::PrivateType:0x00000006219e08 @type_id="null", @value="">

**Comment by seanrucker** [comment]
I have the same problem. Do you have a workaround?

**Comment by diogenes** [comment]
Maybe it helps to clarify: http://lost-in-code.blogspot.com/2011/10/yaml-in-ruby-19.html

**Comment by espen** [comment]
@diogenes Yeah. Perhaps taps should specify psych as the YAML engine. YAML::ENGINE.yamler = "psych"
